frontend/src/pages/Player/Toolbar/DevToolsWindowV2/NetworkPage/NetworkPage.tsx

```diff
@@ -446,6 +470,9 @@ const ResourceRow = ({
 						  })
 						: MillisToMinutesAndSeconds(resource.startTime)}
 				</Text>
+				<Text size="small" weight={showingDetails ? 'bold' : 'medium'}>
+					{formatTime(resource.responseEnd - resource.startTime)}
```

Comment:
Correct me if I"m wrong, but shouldn't `resource.startTime` be `resource.responseStart`?

Reply:
@aaronmaxcarver , yes, a new column sounds good. Though, maybe this is a better fit for a new PR? Will defer to @Vadman97 on exact definitions.

Reply:
@jay-khatri Yeah new column was requested per the original issue, https://github.com/highlight/highlight/issues/4456#issue-1605911039. 

The question is do we want `latency` or do we want `duration`? What do you want to show the user?

They are not the same value, so I don't know which one to calculate and display. Can you provide guidance on which one you want to show users? Then I'll follow @Vadman97 guidance on the proper definition.

Side note -the value `resource.responseEnd - resource.startTime` is already being display in the right hand panel (visible after clicking a request row). We call it 'time' currently. I'll update its name to 'Duration'.

Reply:
for latency, i suggest `resource.responseEnd - resource.responseStart`
for duration, `resource.responseEnd - resource.startTime` is what we want since that captures the entire actual request send/response (ignoring having the request data getting rendered via the dom events)

i think it would be a bit busy to include both `latency` and `duration` though. i think we should just have a column for `duration` and revisit a more granular breakdown in the future (if we want to mimic the full google chrome dev tools flame graph at some point)